### ISSUE TITLE
Remote browser stops on the last page of the test run once task is finished

### ISSUE THREAD

**Issue by inikulin** [DevExpress/testcafe#162]
If you use remote browser for the testrun via `testcafe remote some-test.js` then test is completed remote just frozes on the last page of the test run. I suppose this happens because we terminate process before remote goes to the idle page. However, I think we should wait till remote rich the idle page and shown disconnection status there.

**Comment by AndreyBelym** [comment]
I'm working on it.

**Comment by AndreyBelym** [comment]
The issue has become a duplicate of #131.
